src/_guides/libraries/library-tour_migrated.md

```diff
@@ -443,12 +443,15 @@ assert(ingredients.length == 3);
 // Remove an item from a set.
 ingredients.remove('gold');
 assert(ingredients.length == 2);
+
+// You can also create sets using a literal.
```

Comment:
Should we have the literal at the top?

Also... It seems weird to not have the generic type for the Set. Shouldn't we model specifying the type? This might be an issue in other code samples, too. A quick look finds the questionable looking `var teamAssignments = {};` and I'm sure there are more.

Reply:
I think here it makes more sense to start with the `Set` constructor:

* There is no empty set literal (unless you pair it with generics). `{}` means an empty Map.
* We're talking about the library, and its classes — not language features. If I could, I would start the `List` section with `List()` as well, but it's not possible.

As for the generic type: Yeah, I wasn't sure about that. The tour seems to take a pretty laissez-faire approach. I wonder if it's remnants from pre-Dart-2?

But, on the other hand, I like to avoid introducing too much stuff at once. Like, when we're looking at the library, and the learner isn't possibly all that familiar with generics, let's keep them away as much as possible.

In _this_ particular case, the generic for `{79, 22, 54}` is inferred. So I wouldn't write in the generic even in production code.

Reply:
> As for the generic type: Yeah, I wasn't sure about that. The tour seems to take a pretty laissez-faire approach. I wonder if it's remnants from pre-Dart-2?


Almost certainly, yes.
 
> But, on the other hand, I like to avoid introducing too much stuff at once. Like, when we're looking at the library, and the learner isn't possibly all that familiar with generics, let's keep them away as much as possible.

I agree with being as simple as possible, but I also don't want to model bad behavior. We could explain the first time we need to use generics, and/or we could link aware to a friendly place where generics are talked about. (https://dart.dev/guides/language/language-tour#generics maybe?)

Reply:
Ok, resolved with latest commit